CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Contributing
+This document describes how to go about contributing to the ISOBlue-Avena repository as well as defining how we manage pull requests, releases, and other collaborative features
+
+
+## General Guidelines
+If you are looking to help us, please visit our [issue tracker](https://github.com/OATS-Group/isoblue-avena/issues). If you would like to suggest or discuss a feature/bug, please email our mailing list found here https://groups.google.com/forum/?oldui=1#!forum/isoblue. Please ensure you mention the 'Avena' version of ISOBlue.
+
+
+## Release Branches and Versioning
+The master branch will track the current unstable version of the codebase (equivalent to the `dev` branch in many other repositories). Stable versions will be maintained using branches in the form of `release/xx.yy.zz`, similar to the [Gitlab Flow](https://docs.gitlab.com/ee/topics/gitlab_flow.html) style. This is primarily for the ability to cherry-pick bug fixes into previous releases. 
+
+Releases will be numbered in the [Semantic Versioning](https://semver.org/) style. Version `XX.YY.ZZ` is major version `XX`, minor version `YY`, and bugfix `ZZ`. Major versions incorporate breaking changes, minor versions introduce new features, and bugfixes... fix bugs
+
+
+## Feature Branches and Pull Requests
+Members of the ISOBlue-Avena repository will use feature/bugfix branches in the form of `feature/feature-name` or `bugfix/bug-name`. Outside contributors forking the repository are encouraged to do the same
```

Comment:
Do we want separate `feature/xxx` and `bugfix/xxx` branches or should they all be under `feature/xxx`?

Reply:
:shrug:

Reply:
I think making the distinction with tags in the branch name is the way to go.

Reply:
@facastiblancor Could you explain that a little more? Do you mean like `feature/ghactions-build-all-docker` for a feature and `feature/ghactions-build-oada-fix` for a bug? Just kinda interpreting the context clues?

Reply:
I mean, it would be better to use the `feature/xxx` prefix for new features and `bugfix/xxx` for fixes on features already on master. I think it is easier to distinguish things using prefixes, rather than having lots of _features_ and scrolling down to tell which one is a new feature and which is a bugfix.

Reply:
Agreed, sounds like I just misunderstood what you meant.